bc_obps/reporting/service/compliance_service.py

```diff
@@ -125,16 +127,17 @@ def get_reporting_only_allocated(report_version_id: int, product_id: int) -> Dec
         reporting_only_allocated = ComplianceService.get_allocated_emissions_by_report_product_emission_category(
             report_version_id, product_id, reporting_only_category_ids
         )
+        return reporting_only_allocated
+
+    @staticmethod
+    def get_unregulated_products_allocated(report_version_id: int) -> Decimal:
         unregulated_product_allocation_records = ReportProductEmissionAllocation.objects.filter(
             report_version_id=report_version_id,
```

Comment:
I think we still need to exclude counting excluded emissions allocated to unregulated products here or we're double counting those emissions.

If I have 10k gsc emissions of which 2k are excluded woody biomass. I have 10k emissions.

I allocate 8k gsc emissions to a regulated product
I allocate 2k gsc emissions to unregulated
I allocate 1k woody biomass emissions to regulated product
I allocate 1k woody biomass emissions to unregulated

I now have 3k unregulated product emissions, 1k reporting-only regulated product emissions & 8k regulated product emissions = 12k.